rpc_jobs/rpc_aio.yml

```diff
@@ -47,8 +54,6 @@
             Cleanup,
             Destroy Slave
           UPGRADE_FROM_REF: "liberty-12.2"
```

Comment:
Yes, agreed - however I'm trying to make minimal changes in this patch. As upgrades are not executed for any branch but mitaka at this stage, these changes can be made in a patch later which enables upgrades for other branches.